ext/standard/basic_functions.c

```diff
@@ -2563,6 +2564,11 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_is_callable, 0, 0, 1)
 	ZEND_ARG_INFO(0, syntax_only)
 	ZEND_ARG_INFO(1, callable_name)
 ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_INFO_EX(arginfo_typeof, 0, 0, 1)
```

Comment:
nit; Use ZEND_RETURN_VALUE instead of 0 in the third arg.  It's better for BC and more readable.